integreat_cms/static/src/js/languages/language-color-fields.ts

```diff
@@ -0,0 +1,20 @@
+const renderLanguageColorPreview = ({ target }: Event) => {
+    const colorSelect = target as HTMLSelectElement;
```

Comment:
To be honest, I don't fully understand the difference between selectedColor and colorSelect. Could you please find a better naming for either one? :)